examples/cloudml-document-ai-patents/get_service_acct.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Create GCP service account, add permissions, and download key.
+
+set -e
+
+# TODO(munn): make remove_quotes() function
```

Comment:
Marking this TODO for followup.

Reply:
Make sure to follow the styleguide on using functions https://google.github.io/styleguide/shell.xml

Reply:
Also see the styleguide bits on [function location](https://google.github.io/styleguide/shell.xml?showone=Function_Location#Function_Location) and [main](https://google.github.io/styleguide/shell.xml?showone=main#main)